packages/components/src/2016-12-01/button.js

```diff
@@ -73,22 +79,24 @@ class Button extends HTMLElement {
     });
   }
 
-  set background(val) {
-    this.state.background = val || this.defaults.background;
+  set background(val = this.defaults.background) {
+    this.state.background = val;
+    this._background = val;
```

Comment:
Why set a private field as well as this.state.background? 

Also this.defaults.background is no longer being set as the default. This seems like due to a bad merge?